src/KeyboardTrait.php

```diff
@@ -41,8 +41,8 @@ public function keyboardPressKeysOnElement($keys, $selector = NULL) {
    */
   public function keyboardPressKeyOnElement($char, $selector = NULL) {
     $driver = $this->getSession()->getDriver();
-    if (!$driver instanceof Selenium2Driver) {
-      throw new UnsupportedDriverActionException('Method can be used only with Selenium driver', $driver);
+    if (!$driver->supportsJavascript()) {
```

Comment:
I believe `supportsJavascript` is on `DriverFactoryInterface` whereas we are referencing an instance of `DriverInterface`

Reply:
Would it not be better to try and evaluate script and just re-throw the provided exception from the driver?
[at this point](https://github.com/drevops/behat-steps/pull/122/files#diff-69de6d47a539ff98196f7e23f1885f58a85dbb803e862e7f3d437a42231a9734L105)

Reply:
Agree that try/catch is better (PR now does that in a few places).

I think KeyboardTrait is a special case which may really depend on Selenium2 providing "Syn", an event synthesizer shim?

Reply:
Removed other changes to KeyboardTrait.php, amended error messages to reference Selenium2 driver since that's the actual intent I believe.